use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java

```diff
@@ -174,8 +215,9 @@ public VehicleRoutePlan build(DemoData demoData) {
                 .limit(demoData.customerCount)
                 .collect(Collectors.toList());
 
-        return new VehicleRoutePlan(name, demoData.southWestCorner, demoData.northEastCorner, tomorrowAt(demoData.vehicleStartTime), tomorrowAt(LocalTime.MIDNIGHT).plusDays(1L), depots, vehicles, customers
-        );
+        return new VehicleRoutePlan(name, demoData.southWestCorner, demoData.northEastCorner,
+                                    tomorrowAt(demoData.vehicleStartTime), tomorrowAt(LocalTime.MIDNIGHT).plusDays(1L),
+                                    depots, vehicles, customers);
```

Comment:
I think your formatting is still wrong; in this case, lines 219 and 220 should be indented, not aligned to the first argument on line 218.

Reply:
Good point! I have reformatted the whole VRP quickstart source code (and kept it as a separate commit for now).

Could you please confirm that the result is expected? Namely, the following two aspects:
- indenting by 4 spaces in HTML,
- using star imports.

Thanks!

Reply:
I don't think we have rules for HTML indenting.
But `*` importsa are forbidden.

Reply:
Thanks! I have removed the last commit with the reformat and reformatted Java code only (so HTML stays as is) without introducing `*` imports.